intro-HPC/ch_introduction.tex

```diff
@@ -142,8 +185,8 @@ \section{What is the \hpcInfra?}
 with \emph{cpuset} support and \emph{BLCR} modules.
 \fi
 \ifleuven
-All the nodes in the \hpc run under the ``\operatingsystem'' operating system,
-version 6.
+All the nodes in the \hpc run under the ``\operatingsystem'' operating system.
+%version 6.
```

Comment:
If you already have it elsewhere, your changes are good, but maybe remove the commented part as well, since this seems to imply information is lost. And it might not be kept up to date, since you will makes further changes of versions  in the macro. Then this comment line might no longer contain correct information.